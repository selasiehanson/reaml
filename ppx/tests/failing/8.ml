let _foo =
 fun [@reaml.componnt "Foo"] () ->
  let _ =
   fun () ->
    let[@reaml] () = () in
    ()
  in
  ()
