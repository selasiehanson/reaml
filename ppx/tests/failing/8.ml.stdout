[%%ocaml.error "[@reaml.componnt] occurs in an inappropriate place"]
